phoenix-core-server/src/main/java/org/apache/phoenix/coprocessor/PagingRegionScanner.java

```diff
@@ -59,8 +70,32 @@ public PagingRegionScanner(Region region, RegionScanner scanner, Scan scan) {
         if (pagingFilter != null) {
             pagingFilter.init();
         }
+
+        TableDescriptor tableDescriptor = region.getTableDescriptor();
+        BloomType bloomFilterType = tableDescriptor.getColumnFamilies()[0].getBloomFilterType();
+        if (bloomFilterType == BloomType.ROW) {
+            // Check if the scan is a multi-point-lookup scan
+            skipScanFilter = ScanUtil.getSkipScanFilter(scan);
```

Comment:
Do we need to remove the skipscan filter from the scan also ?

Reply:
We do not have to remove the skip scan filter and but it makes sense to remove it for simplicity and efficiency (that is one less filter).

Reply:
It turned out that we have to keep the skip scan filter to handle region moves. In that case, we need to do range scan with the skip scan filter to retrieve the next row.

Reply:
We already have some tests that would fail if we remove the filter right?

Reply:
Yes, there are such tests with region moves.